src/main/java/wolox/bootstrap/services/CustomUserDetailService.java

```diff
@@ -22,29 +22,23 @@
     private final UserRepository userRepository;
     private BCryptPasswordEncoder passwordEncoder = new BCryptPasswordEncoder();
 
-
     @Autowired
     public CustomUserDetailService(UserRepository userRepository) {
         this.userRepository = userRepository;
     }
 
-
     @Override
     @Transactional(readOnly = true)
     public UserDetails loadUserByUsername(final String username) throws RuntimeException {
-
-        User user = userRepository.findByUsername(username).orElseThrow(RuntimeException::new);
-
+        final User user = userRepository.findByUsername(username).orElseThrow(RuntimeException::new);
         if (user == null) {
```

Comment:
I know you didn't write this part but i think we can delete this nullcheck 😄  (it can never be null because of the previous search + orelsethrow)